packages/grub2/config/bootargs.cfg

```diff
@@ -1,6 +1,6 @@
 # TODO we could sanity check $partlabel is set here so we can error out before even attempting to boot
 set kernel=/boot/vmlinuz
-if [ ${label} == ${system_label} ]; then
+if [ "${label}" == "${system_label}" ]; then
```

Comment:
I got an unexpected behavior when system_label was an empty string, the comparison was always returning true. Using quotes just fixed it.